includes/class-social.php

```diff
@@ -272,4 +272,19 @@ protected function print_label( array $field ) {
 
 	}
 
+	/**
+	 * Is the social media selection deprecated
+	 *
+	 * @param  array  $field Field array
+	 *
+	 * @return boolean
+	 *
+	 * @since NEXT
+	 */
+	protected function is_social_media_deprecated( $field ) {
```

Comment:
@EvanHerman Having a method for this is overkill IMO.

And `$this->is_social_media_deprecated( $field )` is longer than `! empty( $field['deprecated'] )`.